cli/drivers/WordpressComposerInstallValetDriver.php

```diff
@@ -0,0 +1,105 @@
+<?php
+
+class WordpressComposerInstallValetDriver extends ValetDriver
+{
+    /**
+     * Determine if the driver serves the request.
+     *
+     * @param  string  $sitePath
+     * @param  string  $siteName
+     * @param  string  $uri
+     * @return void
+     */
+    public function serves($sitePath, $siteName, $uri)
+    {
+        if (file_exists($composerFile = $sitePath . '/composer.json')) {
+            if (file_exists($sitePath . '/wp-config.php')) {
+                return true;
+            }
+
+            $composerJson = json_decode(file_get_contents($composerFile));
```

Comment:
`json_decode()` on every request seems a bit inefficient to me.

How about `is_dir($sitePath . '/vendor/johnpbloch/wordpress-core-installer/')` instead?